tidb-in-kubernetes/recovery-after-accidental-deletion.md

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
Please rename this file to `tidb-in-kubernetes.md` and move it to `/dev/how-to/troubleshoot/tidb-in-kubernetes.md`.